client-app/src/desktop/tabs/examples/ExamplesTab.js

```diff
@@ -1,138 +1,230 @@
-import {hoistCmp} from '@xh/hoist/core';
-import {a, code, hbox, vframe} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistModel, managed, useLocalModel} from '@xh/hoist/core';
+import {a, code, div, filler, hframe, iframe} from '@xh/hoist/cmp/layout';
 import {button} from '@xh/hoist/desktop/cmp/button';
 import {Icon} from '@xh/hoist/icon';
-import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {panel, PanelModel} from '@xh/hoist/desktop/cmp/panel';
+import {toolbar} from '@xh/hoist/desktop/cmp/toolbar';
 
 import './ExamplesTab.scss';
-import {wrapper} from '../../common';
 import {ToolboxLink} from '../../../core/cmp/ToolboxLink';
+import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {directoryPanel} from '../../../examples/contact/DirectoryPanel';
+import {todoPanel} from '../../../examples/todo/TodoPanel';
+import {recallsPanel} from '../../../examples/recalls/RecallsPanel';
+import {fileManager} from '../../../examples/filemanager/FileManager';
+import {newsPanel} from '../../../examples/news/NewsPanel';
+import {portfolioPanel} from '../../../examples/portfolio/PortfolioPanel';
+
 
 export const examplesTab = hoistCmp.factory(
-    () => wrapper(
-        hbox({
-            className: 'example-tile-container',
-            flexWrap: 'wrap',
-            items: getExamples().map(e => panel({
+    () => {
+        const impl = useLocalModel(LocalModel);
```

Comment:
Make this LocalModel a named model "ExampleTabModel" and promote it to be the actual model for the tab.  This is our new best practice and will allow you to avoid having to pass it explicitly to your sideBar and appTileBar